docs/core/sdk.md

```diff
@@ -34,6 +34,11 @@ interested in how to set up the SDK on your CI build server, see [Use the .NET S
 By default, the SDK installs in a "side-by-side" (SxS) manner, which means multiple versions
 can coexist at any given time on a single machine. For information about how the version gets picked when you're running CLI commands, see [Select the .NET version to use](versions/selection.md).
 
+## Security guide
+
+> [!IMPORTANT]
+> The .NET SDK locates and executes various tools that ship as separate executable binaries in the SDK. In most cases, the SDK is able to determine the full path to the executable. However, there are exceptions where the SDK cannot determine the path and relies on user input. It's possible that a user can provide input that causes the .NET SDK to execute malicious software. For this reason, you shouldn't trust any repos with binaries that match specific names in the root folder, for example, *msbuild.exe*.
```

Comment:
I'd like to be more explicit about what the "root folder" means if you can help clarify that.

Reply:
Perhaps "SDK Install location" instead, with an example like `C:\Program Files\dotnet\sdk\7.0.400\` and/or `/usr/bin/share/dotnet/sdk/7.0.400`? The main point is that the 'root folder' they should be comparing against is the per-SDK-versioned folder.